adapter/README.md

```diff
@@ -9,6 +9,10 @@ with the wiring when using the `rpi-rgb-led-matrix` code.
      buffer the outputs using the ...
    * [Active-3](./active-3) board. Supports three parallel chains with active buffering
      and 3.3V -> 5V level shifting for best reliability. Requires SMD soldering.
+     As another option you can buy it from these locations:
+     * https://www.electrodragon.com/product/rgb-matrix-panel-drive-board-raspberry-pi/
+       ($3/board, but fairly long and/or expensive shipping from HKG)
```

Comment:
Please add a "Not affiliated" disclaimer, as I don't want to give support for their product.

Reply:
Fair point, done.